spyder/app/tour.py

```diff
@@ -815,6 +832,11 @@ def mousePressEvent(self, event):
 #            if clicked_widget == self.label_current:
 #            self.context_menu_requested(event)
 
+    def focusOutEvent(self, event):
+        """Reimplement Qt method."""

```

Comment:
Ok, no prob. @mariacamilaremolinagutierrez, please change *Reimplement* for *Override* in all docstrings you introduced :-)